inttest/Makefile.variables

```diff
@@ -45,3 +45,4 @@ smoketests := \
 	check-customdomain \
 	check-capitalhostnames \
 	check-customca \
+	check-dualstack-dynamicconfig \
```

Comment:
This list is getting unwieldy. Maybe we should start to order things alphabetically here, instead of adding stuff at the end.